test/integration/error.test.js

```diff
@@ -34,6 +38,9 @@ describe(Support.getTestDialectTeaser('Sequelize Errors'), function () {
       expect(validationError).to.be.instanceOf(Sequelize.ValidationError);
       expect(validationError).to.be.instanceOf(Error);
       expect(validationError).to.have.property('name', 'SequelizeValidationError');
+      console.log(validationError.message);
+      console.log(typeof(validationError.message));
+      expect(validationError.message).to.be('notNull Violation: <field name> cannot be null,\nstring violation: <field name> cannot be an array or an object');
```

Comment:
`TypeError: Property 'be' of object [object Object] is not a function`

I think `to.match` is what you are looking for. Also, please squashs your commits into one http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html